leftover debugging code makes build of 2022_03_3 tarball fail

hello
just tried to build RDKit_2022_03_3.tgz and got:
```
/usr/bin/ld: Code/GraphMol/FileParsers/CMakeFiles/testMultithreadedMolSupplier.dir/testMultithreadedMolSupplier.cpp.o: in function `testSmiOld(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, int, int, bool, bool, unsigned int, bool)':
testMultithreadedMolSupplier.cpp:(.text+0x4cd): undefined reference to `PrintThread::cout_mutex'
/usr/bin/ld: testMultithreadedMolSupplier.cpp:(.text+0x50b): undefined reference to `PrintThread::cout_mutex'
/usr/bin/ld: Code/GraphMol/FileParsers/CMakeFiles/testMultithreadedMolSupplier.dir/testMultithreadedMolSupplier.cpp.o: in function `testSDOld(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, bool, bool, bool, unsigned int, bool)':
testMultithreadedMolSupplier.cpp:(.text+0x9b4): undefined reference to `PrintThread::cout_mutex'
/usr/bin/ld: testMultithreadedMolSupplier.cpp:(.text+0x9f2): undefined reference to `PrintThread::cout_mutex'
/usr/bin/ld: testMultithreadedMolSupplier.cpp:(.text+0xb14): undefined reference to `PrintThread::cout_mutex'
/usr/bin/ld: Code/GraphMol/FileParsers/CMakeFiles/testMultithreadedMolSupplier.dir/testMultithreadedMolSupplier.cpp.o:testMultithreadedMolSupplier.cpp:(.text+0xb52): more undefined references to `PrintThread::cout_mutex' follow
collect2: error: ld returned 1 exit status
```

It looks like there is a debugging code fragment leftover in testMultithreadedMolSupplier.cpp
As a matter of fact, this allows the build to complete successfully for me:
```
diff -c Code/GraphMol/FileParsers/testMultithreadedMolSupplier.cpp.FIX_TESTMULTITHREADEDMOLSUPPLIER_CPP Code/GraphMol/FileParsers/testMultithreadedMolSupplier.cpp
*** Code/GraphMol/FileParsers/testMultithreadedMolSupplier.cpp.FIX_TESTMULTITHREADEDMOLSUPPLIER_CPP	Fri Jun  3 15:15:47 2022
--- Code/GraphMol/FileParsers/testMultithreadedMolSupplier.cpp	Fri Jun  3 15:15:47 2022
***************
*** 27,41 ****
  using namespace RDKit;
  using namespace std::chrono;
  
! // thread safe printing for debugging
! // Usage Example: PrintThread{} << "something";
! struct PrintThread : public std::stringstream {
!   static std::mutex cout_mutex;
!   ~PrintThread() override {
!     std::lock_guard<std::mutex> l{cout_mutex};
!     std::cout << rdbuf();
!   }
! };
  
  void testSmiConcurrent(std::istream *strm, bool takeOwnership,
                         std::string delimiter, int smilesColumn, int nameColumn,
--- 27,41 ----
  using namespace RDKit;
  using namespace std::chrono;
  
! // // thread safe printing for debugging
! // // Usage Example: PrintThread{} << "something";
! // struct PrintThread : public std::stringstream {
! //   static std::mutex cout_mutex;
! //   ~PrintThread() override {
! //     std::lock_guard<std::mutex> l{cout_mutex};
! //     std::cout << rdbuf();
! //   }
! // };
  
  void testSmiConcurrent(std::istream *strm, bool takeOwnership,
                         std::string delimiter, int smilesColumn, int nameColumn,

```

Building with:
```
-DCMAKE_INSTALL_PREFIX=/usr
-DRDK_INSTALL_INTREE=OFF
-DLIB_SUFFIX=64
-DPYTHON_INSTDIR=/usr/python3/site_python3/lib64
-DCMAKE_VERBOSE_MAKEFILE=ON
-DBoost_INCLUDE_DIR:PATH=/usr/include
-DPYTHON_EXECUTABLE:FILEPATH=/usr/python3/bin/python
-DRDK_BOOST_PYTHON3_NAME=python310
-DPYTHON_NUMPY_INCLUDE_PATH=/usr/python3/site_python3/lib64/numpy.egg/numpy/core/include
-DBoost_NO_BOOST_CMAKE=ON
```

thanks for your valuable work
ciao
-gabriele

--- a/Code/GraphMol/FileParsers/testMultithreadedMolSupplier.cpp
+++ b/Code/GraphMol/FileParsers/testMultithreadedMolSupplier.cpp
@@ -30,7 +30,7 @@ using namespace std::chrono;
 // thread safe printing for debugging
 // Usage Example: PrintThread{} << "something";
 struct PrintThread : public std::stringstream {
-  static std::mutex cout_mutex;
+  inline static std::mutex cout_mutex;
   ~PrintThread() override {
     std::lock_guard<std::mutex> l{cout_mutex};
     std::cout << rdbuf();

